README.md

```diff
@@ -216,6 +216,7 @@ for {
 ```
 
 For complete usage of go-github, see the full [package docs][].
+Also, take a look at [documentation][].
```

Comment:
Let's please delete this line since they are essentially pointing to the same thing... the package docs.
Then let's please change line 225 below to change the "package docs" to point to the new documentation as "godoc.org" will be going away.